ui/apps/platform/webpack.ocp-plugin.config.js

```diff
@@ -59,7 +61,7 @@ const config = {
                 use: ['style-loader', 'css-loader'],
             },
             {
-                test: /\.(png|jpg|jpeg|gif|svg|woff2?|ttf|eot|otf)(\?.*$|$)/,
+                test: /\.(png|jpg|jpeg|gif|svg|woff2?|ttf|eot|otf|ico)(\?.*$|$)/,
```

Comment:
For my understanding, was `ico` an unintended omission, or is it to support images specific to plugin?